Feature/basic

jhlee343 opened the pull request:
- user, follow, review , reviewReport, warning 레포지터리 디렉터리 이름명 수정
- 사용자 식사권 리스트 불러오기 기능 구현
- 낙찰받은 경매 불러오기 
  1. 식사 전 
  2. 식사 후
  구분해서 구현
- vip 소개 불러오기 기능 구현

- follow , warning entity  변수명 수정, vipinfo  id 타입 수정

- editInfo 구현 예정

jhlee343 committed:
Feat : successAuctionListDto

jhlee343 committed:
Merge branch 'develop' of https://github.com/jhlee343/Back-end into feature/basic

jhlee343 committed:
Feat : userAuctionParticipateListDto

jhlee343 committed:
Merge branch 'develop' of https://github.com/jhlee343/Back-end into feature/basic

jhlee343 committed:
Feat : vipInfo 불러오기 api

jhlee343 committed:
Feat : editInfo api

jhlee343 committed:
Merging Conflit

jhlee343 committed:
Feat : load ticketList api

jhlee343 committed:
Rename : userRepo file dir 수정

jhlee343 committed:
Rename : follow Repo dir 수정

jhlee343 committed:
conflict

jhlee343 committed:
merging conflict

jhlee343 committed:
Feat : auctionSuccessList api ã개발

jhlee343 committed:
Refactor : ticketList

jhlee343 committed:
Merge branch 'develop' of https://github.com/jhlee343/Back-end into feature/basic

jhlee343 committed:
Rename : user, follow, review, reviewReport, vip, warning dir 이름 수정

jhlee343 committed:
Refactor : follow entity 수정

Minjeong63 commented:
확인했습니다

dnjs2721 commented:
api 개발이 많아 추적이 좀 힘드네요..
vipinfo는 vip 마이페이지에서 조회하는 api 맞을까요
그렇다면 evidence는 필요없어보입니다.

dnjs2721 commented:
혹시 페이지 쿼리 만드실 때 리스트 타입 쿼리도 만드시는거 같은데 이유가 있을까요?
다른 레포에서도 그렇게 하셨던거 같아서요

jhlee343 commented:
저번개발때 페이지에서 리스트로 바뀐거 생각나서 혹시몰라 둘다 적어두고있습니다..
URL같은 경우는 제외시키겠습니다.

dnjs2721 commented:
리뷰 조회시 사용자 uuid 를 반환하는데 사용자 닉네임을 넘겨줘야하지 않을까요?
추가로 BasicController ->BasicUserController로 수정 바랍니다. 서비스, 레포도 동일합니다.
이름만 보고 어떤 파일인지 보기 힘듭니다.

dnjs2721 commented:
더 있는것 같은데 찾기가 힘드네요. 나머지는 나중에 코드로 확인하겠습니다.....

jhlee343 committed:
Refactor : vipInfo evidence url remove,
send review , receive review nickname반환으로 변경,
basicUs´련 controller, service basicUser로 이름변경

dnjs2721 commented:
확인했습니다
